che-eclipse-license-resource-bundle/pom.xml

```diff
@@ -2,13 +2,11 @@
 <!--
 
     Copyright (c) 2012-2018 Red Hat, Inc.
-    All rights reserved. This program and the accompanying materials
-    are made available under the terms of the Eclipse Public License v1.0
-    which accompanies this distribution, and is available at
-    http://www.eclipse.org/legal/epl-v10.html
+    This program and the accompanying materials
```

Comment:
Maybe it would be better not to remove `All rights reserved.` prefix?

Reply:
https://www.eclipse.org/legal/epl-2.0/faq.php#h.q72cnghf29k0

Reply:
@benoitf Thanks for sharing